xboxpy/interface/if_xbdm.py

```diff
@@ -123,11 +123,19 @@ def GetMem(addr, length):
 
 def SetMem(addr, data):
   value = bytes(data)
-  if True:
+
+  # Unfortunately XBDM has a maximum line-buffer of about ~500 symbols.
+  # We'll be conservative and only allow 200 bytes (400 hex digits).
+  # That leaves ~100 bytes for the actual command.
+  max_len = 200
+  if len(value) > max_len:
+    SetMem(addr, value[0:max_len])
```

Comment:
Personally would prefer to re-write this to an iterative solution over a recursive one.

Reply:
Indifferent about this. This is more of a last-minute addition to the PR as I ran into the issue while testing.
I can also throw together an iterative solution if necessary.

What I liked about this one was that I was able to avoid `value[offset:offset+chunk_len]` syntax which is harder to read in my opinion. I'd probably also need to extra variables for `offset` and `len`?

Reply:
You can have `begin` and `end` vars or something if you want.. something like `buffer[begin:end]`.

Up to you, otherwise lgtm.

Reply:
I came up with

```python
  begin = 0
  end = len(value)
  while begin < end:

    cmd="setmem addr=0x" + format(addr, 'X') + " data="
    for i in range(begin, min(begin + max_len, end)):
      cmd += format(value[i], '02X')
    xbdm_command(cmd)

    begin += max_len
```

However, I think it's harder to read (and untested).
Will probably merge as-is.